airbyte-integrations/connectors/source-zoom/source_zoom/manifest.yaml

```diff
@@ -206,15 +213,16 @@ definitions:
         type: SubstreamPartitionRouter
         parent_stream_configs:
           - stream: "#/definitions/meetings_list_tmp_stream"
-            parent_key: "uuid"
+            parent_key: "id"
```

Comment:
The Zoom docs state that either a uuid or id can be provided. However, in my tests using the "uuid" in requests simply returned a "Meeting does not exist" message. By changing to "id" I was able to successfully fetch the records in our account.